Mathlib/Algebra/GeomSum.lean

```diff
@@ -256,58 +256,58 @@ protected theorem Commute.geom_sum₂_comm {α : Type u} [Semiring α] {x y : α
   refine Finset.sum_congr rfl fun i hi => ?_
   simpa [Nat.sub_sub_self (Nat.succ_le_succ_iff.mp (Finset.mem_range.mp hi))] using h.pow_pow _ _
 
-theorem geom_sum₂_comm {α : Type u} [CommSemiring α] (x y : α) (n : ℕ) :
+theorem geom_sum₂_comm {R : Type u} [CommSemiring R] (x y : R) (n : ℕ) :
     ∑ i ∈ range n, x ^ i * y ^ (n - 1 - i) = ∑ i ∈ range n, y ^ i * x ^ (n - 1 - i) :=
   (Commute.all x y).geom_sum₂_comm n
 
-protected theorem Commute.geom_sum₂ [DivisionRing α] {x y : α} (h' : Commute x y) (h : x ≠ y)
+protected theorem Commute.geom_sum₂ [DivisionRing R] {x y : R} (h' : Commute x y) (h : x ≠ y)
```

Comment:
Can this one be
```suggestion
protected theorem Commute.geom_sum₂ [DivisionRing 𝕜] {x y : R} (h' : Commute x y) (h : x ≠ y)
```
or similar?

Reply:
I'll search for division rings in the diff. Thanks for catching this!

Reply:
Done.